<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget#Registration {
      background-color: #f5f5f5;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   
   <!-- Заголовок -->
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Создать аккаунт</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">
       font-size: 28px;
       font-weight:  bold;
       color: #333;
       margin-bottom: 5px;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="text">
      <string>Заполните данные для регистрации</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">
       font-size: 13px;
       color: #666;
       margin-bottom: 15px;
      </string>
     </property>
    </widget>
   </item>
   
   <!-- Форма -->
   <item>
    <widget class="QFrame" name="formFrame">
     <property name="styleSheet">
      <string notr="true">
       QFrame#formFrame {
         background-color: white;
         border-radius: 10px;
       }
      </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="formLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      
      <!-- Имя -->
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Имя: </string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #333; font-weight: 500;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="placeholderText">
         <string>Введите ваше имя</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
            border: 2px solid #ddd;
            border-radius:  5px;
            padding:  5px 10px;
            font-size: 14px;
            background-color: white;
          }
          QLineEdit:focus {
            border-color: #2196F3;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noName">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px; margin-bottom: 5px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      
      <!-- Фамилия -->
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фамилия:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:  14px; color: #333; font-weight: 500;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="surnameEdit">
        <property name="placeholderText">
         <string>Введите вашу фамилию</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
            border: 2px solid #ddd;
            border-radius: 5px;
            padding: 5px 10px;
            font-size:  14px;
            background-color: white;
          }
          QLineEdit:focus {
            border-color: #2196F3;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noSurname">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px; margin-bottom: 5px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      
      <!-- Email -->
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Email:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #333; font-weight: 500;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailEditReg">
        <property name="placeholderText">
         <string>Введите ваш email</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
            border: 2px solid #ddd;
            border-radius: 5px;
            padding: 5px 10px;
            font-size: 14px;
            background-color: white;
          }
          QLineEdit:focus {
            border-color: #2196F3;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noEmail">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="checkEmail">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px; margin-bottom: 5px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      
      <!-- Пароль -->
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Пароль: </string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #333; font-weight: 500;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="passwordLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLineEdit" name="passwordEditReg">
          <property name="placeholderText">
           <string>Введите пароль</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">
            QLineEdit {
              border: 2px solid #ddd;
              border-radius: 5px;
              padding: 5px 10px;
              font-size: 14px;
              background-color: white;
            }
            QLineEdit:focus {
              border-color: #2196F3;
            }
           </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showPasswordReg">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string></string>
          </property>
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
              background-color: #f5f5f5;
              border: 2px solid #ddd;
              border-radius: 5px;
            }
            QPushButton:hover {
              background-color: #e0e0e0;
            }
            QPushButton: pressed {
              background-color:  #E3F2FD;
              border-color: #2196F3;
            }
           </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="passwordRequirements">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px; margin-bottom: 5px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      
      <!-- Повторите пароль -->
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Повторите пароль:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px; color: #333; font-weight: 500;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEditRegAgain">
        <property name="placeholderText">
         <string>Повторите пароль</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
            border: 2px solid #ddd;
            border-radius: 5px;
            padding: 5px 10px;
            font-size: 14px;
            background-color: white;
          }
          QLineEdit:focus {
            border-color: #2196F3;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passwordsNotMatch">
        <property name="styleSheet">
         <string notr="true">color: #f44336; font-size: 12px; margin-bottom: 10px;</string>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      
      <!-- Кнопка регистрации -->
      <item>
       <widget class="QPushButton" name="regButton">
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            background-color: #4CAF50;
            color:  white;
            border: none;
            border-radius: 5px;
            font-size: 16px;
            font-weight: bold;
          }
          QPushButton:hover {
            background-color: #45a049;
          }
          QPushButton:pressed {
            background-color: #2E7D32;
          }
          QPushButton:disabled {
            background-color: #ccc;
          }
         </string>
        </property>
       </widget>
      </item>
      
      <!-- Разделитель -->
      <item>
       <widget class="QFrame" name="separator">
        <property name="frameShape">
         <enum>QFrame::Shape:: HLine</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #ddd; margin: 10px 0;</string>
        </property>
       </widget>
      </item>
      
      <!-- Кнопка назад -->
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Уже есть аккаунт?  Войти</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
            background-color: #2196F3;
            color:  white;
            border: none;
            border-radius: 5px;
            font-size:  14px;
            font-weight:  bold;
          }
          QPushButton:hover {
            background-color: #1976D2;
          }
          QPushButton:pressed {
            background-color: #0D47A1;
          }
         </string>
        </property>
       </widget>
      </item>
      
     </layout>
    </widget>
   </item>
   
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>surnameEdit</tabstop>
  <tabstop>emailEditReg</tabstop>
  <tabstop>passwordEditReg</tabstop>
  <tabstop>passwordEditRegAgain</tabstop>
  <tabstop>regButton</tabstop>
  <tabstop>backButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
